browser-test/src/admin_program_translation.test.ts

```diff
@@ -164,6 +164,37 @@ describe('Admin can manage translations', () => {
     await endSession(browser);
   });
 
+  it('deleting help text in question edit view deletes all help text translations', async () => {
+    const { browser, page } = await startSession();
+
+    await loginAsAdmin(page);
+    const adminQuestions = new AdminQuestions(page);
+
+    // Add a new question with help text
+    const questionName = 'translate-help-text-deletion';
+    await adminQuestions.addNumberQuestion(questionName);
+
+    // Add a translation for a non-English language.
+    await adminQuestions.goToQuestionTranslationPage(questionName);
+    const adminTranslations = new AdminTranslations(page);
+    await adminTranslations.selectLanguage('Spanish');
+    await adminTranslations.editQuestionTranslations('something different', 'help text different');
+
+    // Edit the question and delete the help text.
+    await adminQuestions.changeQuestionHelpText(questionName, "");
+
+    // Edit the question and add help text back
+    await adminQuestions.changeQuestionHelpText(questionName, "a new help text");

```

Comment:
question: why add it back?

Reply:
Its a way to check that the translations for the old help text are properly deleted. Otherwise there may be a case where:

1. create question with help text "aaa"
2. add spanish translation "aaa"
3. delete help text
4. add help text "bbb"

At this point we expect no translations to be present. It would be a bug if it preserved the "aaa" translation.

Reply:
Thanks for the explanation!